OsmAnd/src/net/osmand/plus/search/listitems/QuickSearchListItem.java

```diff
@@ -102,6 +102,10 @@ public static String getName(OsmandApplication app, SearchResult searchResult) {
 			case LOCATION:
 				LatLon latLon = searchResult.location;
 				return PointDescription.getLocationNamePlain(app, latLon.getLatitude(), latLon.getLongitude());
+			case POI:
+				if (((Amenity)searchResult.object).isClosed()) {
+					return searchResult.localeName + " (Closed)";
```

Comment:
Very bad, please use translate string, it must be something R.string.shared_string_closed